.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: false
+
+language: java
```

Comment:
Why do we use java? Just kidding, looks great other than aforementioned 7 and 8 issue.